examples/send_interval.rs

```diff
@@ -14,7 +14,7 @@ struct Greeter;
 impl Actor for Greeter {
     type Stop = ();
 
-    async fn stopped(self) -> Self::Stop {}
+    async fn stopped(self) {}
```

Comment:
Seeing that CI doesn't break with this change, means we are lacking a test or an example that demonstrates custom `Stop` values!

Reply:
> Is this necessary? It would be unfortunate if we lose this feature, also because the associated type is now useless?

No feature is lost here, this is just in the example where `Stop` is already ()

Reply:
> 

Seeing that CI doesn't break with this change, means we are lacking a test or an example that demonstrates custom Stop values!

A test could be good regardless!

Reply:
> > Is this necessary? It would be unfortunate if we lose this feature, also because the associated type is now useless?
> 
> No feature is lost here, this is just in the example where `Stop` is already (), so I just removed it because `-> Self::Stop` is equivalent to `-> ()` which can be left out

Ah my bad, I should have looked more closely! I thought this is removing the return value from the trait!